tests/unit/test_lda_modeling.py

```diff
@@ -0,0 +1,42 @@
+
+import gensim
+import numpy
+import spacy
+
+from sem_covid.services.sc_wrangling.data_cleaning import clean_text_from_specific_characters
+from sem_covid.entrypoints.notebooks.topic_modeling.topic_modeling_wrangling.lda_modeling import WordsModeling
+
+nlp = spacy.load("en_core_web_sm")
+
+
+def test_word_modeling(transformed_pwdb_dataframe):
```

Comment:
you perform the whole topic modellign here, not testing a specific function. Or you do? If sho what do you test here? 

what will happen when we change the corpus? or the language model?